ModelBundle/Document/AbstractStatus.php

```diff
@@ -29,7 +29,7 @@
     protected $name;
 
     /**
-     * @ODM\EmbedMany(targetDocument="OpenOrchestra\ModelInterface\Model\TranslatedValueInterface")
+     * @ODM\EmbedMany(targetDocument="OpenOrchestra\ModelInterface\Model\TranslatedValueInterface", strategy="set")
```

Comment:
i think they are also a bug in a setter, you can see an example in document content with parameter attributes